hazelcast/src/main/java/com/hazelcast/client/config/ClientCloudConfig.java

```diff
@@ -89,4 +89,11 @@ public int hashCode() {
         result = 31 * result + (enabled ? 1 : 0);
         return result;
     }
+
+    @Override
+    public String toString() {
+        return "ClientCloudConfig{"
+                + "enabled=" + enabled
```

Comment:
why not print `discoveryToken` as well?

Reply:
I discussed it with cloud team and they prefer not to display it. So, I am going to leave this as it is